docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -96,6 +96,11 @@ To learn more about each related process, select the process in the left panel o
 * The `process-pid`
 * The user name and domain that ran the process
 * Any other relevant process information
+* Any associated alerts
++
+preview::[]
++
+NOTE: This feature requires a Platinum license, and you must add the `xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']` feature flag to the `kibana.yml` file.
```

Comment:
A few things here for clarification: 
- I think we need to specify which feature requires the platinum license, aka the "this feature". 
- I made this one pretty much identical to Joe's comment below. 
- Making sure you meant to put this twice in the doc. 
```suggestion
NOTE: This feature requires a https://www.elastic.co/pricing[Platinum license]. In addition, to display it in {elastic-sec}, you must add the following feature flag to the `kibana.yml` file: `xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']` .
```

Reply:
@kqualters-elastic @michaelolo24 is there an official feature name for this? Otherwise, I was thinking about referring to it as "the ability to expose alerts in the event analyzer" or "the functionality that shows alerts in the process tree". Here's an example:

 

> NOTE: The functionality that shows alerts in the process tree requires a https://www.elastic.co/pricing[Platinum license]. In addition, to display it in {elastic-sec}, you must add the following feature flag to the `kibana.yml` file: `xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']` .

Reply:
@kqualters-elastic and I came up with the following way of describing/naming this feature in the docs:

> NOTE: Displaying alerts in the process tree requires a https://www.elastic.co/pricing[Platinum license].

@paulewing does this work or would you prefer that it be described in a different way?

Reply:
@jmikell821 to answer your last question, I was thinking about removing the first instance of the tech preview tag and this note. The info is provided later and is more relevant to the section it was added to than this list.